### ISSUE TITLE
Sound all delayed and not stopping on release

### ISSUE THREAD

**Issue by KlutzyBubbles** [Etuldan/MidiControl#36]
Had this program working perfectly for about 2 weeks then today it randomly seemed to add a huge delay after clicking a sound with no way to stop it (input the stop button because only 1 button can be clicked at a time).

It seems to play the sound fine, then wait ~5-10 seconds then play the next sound that was clicked so on until it has played all the buttons that have been clicked, this can be 2 inputs or 20.

I didnt change anything from last night until today, so i thought it was a version thing so i updated and still the same thing. BUT now after the update whenever i go to edit a sound the GUI freezes and i have to force quit it (but the midi inputs still work with that delay), this is also the case for the tray menu, completely unresponsive.

I do use other programs that listen to midi inputs on the same device but again for 2 weeks this was never a problem.

please let me know the best way to debug this as i would like to find the reason for it (if possible) rather than restarting my pc and hoping it goes away.

**Comment by Etuldan** [comment]
Maybe related to #32

**Comment by Etuldan** [comment]
Could you send me your profile file (default is keybinds.json located in `%Appdata%\midicontrol\`) ? @KlutzyBubbles

**Comment by KlutzyBubbles** [comment]
> Could you send me your profile file (default is keybinds.json located in `%Appdata%\midicontrol\`) ? @KlutzyBubbles

```json
{"Hey Wake Up":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\you_were_poked.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":16,"Channel":1,"Input":0},"Shoot":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\spas-shot-1.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":18,"Channel":1,"Input":0},"Reload":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\shotgun-reload-old_school-ra_the_sun_god-580332022.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":17,"Channel":1,"Input":0},"Mute":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":null,"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":{"StopAllSound":false,"SwitchToProfile":"Default"},"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":109,"Channel":1,"Input":1},"User 2":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":null,"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":{"StopAllSound":false,"SwitchToProfile":"test"},"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":110,"Channel":1,"Input":1},"Anime Wow":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\anime-wow-sound-effect.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":45,"Channel":1,"Input":0},"ahhh":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\aughhhhh-aughhhhh.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":15,"Channel":1,"Input":0},"Bonk":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\bonk.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":28,"Channel":1,"Input":0},"run":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\run-vine-sound-effect.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":27,"Channel":1,"Input":0},"discoird":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\discord-sounds.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":26,"Channel":1,"Input":0},"fake news":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\donald-trump-fake-news-sound-effect.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":25,"Channel":1,"Input":0},"fbi":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\FBI open up Sound Effect.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":38,"Channel":1,"Input":0},"fart reverb":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\fart-with-reverb.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":37,"Channel":1,"Input":0},"tayhya":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\they-ask-you-how-you-are-and-you-just-have-to-say-that-youre-fine-sound-effect_IgYM1CV.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":36,"Channel":1,"Input":0},"quack":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\mac-quack.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":35,"Channel":1,"Input":0},"mcdonals":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\mcdonalds-beeping-sound.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":false,"Loop":true,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":48,"Channel":1,"Input":0},"Stop all":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"","DeviceName":"","StopWhenReleased":false,"Loop":false,"Volume":1.0,"StopAllOtherSounds":true,"Device":-1},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":19,"Channel":1,"Input":0},"no bullshit":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\no-1-bullshit-guy.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":43,"Channel":1,"Input":0},"pew pew":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\pew_pew-dknight556-1379997159.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":41,"Channel":1,"Input":0},"police":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\police-beep.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":44,"Channel":1,"Input":0},"pan":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\pubg-pan-sound-effect.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":42,"Channel":1,"Input":0},"huh":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\videoplayback_y6EZG5Z.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":46,"Channel":1,"Input":0},"vine boom":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\vine-boom-sound-effect_KT89XIq.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":51,"Channel":1,"Input":0},"windows error":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\windows-10-error-sound.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":52,"Channel":1,"Input":0},"THOMAS":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\loud-version_7n1qEm2.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":47,"Channel":1,"Input":0},"Sex":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\Voicy_Loud moan (BASS BOOSTED).mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":34,"Channel":1,"Input":0},"Fortnite":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\fortnite-default-dance-bass-boosted.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":24,"Channel":1,"Input":0},"What meme":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\what-bottom-text-meme-sanctuary-guardian-sound-effect-hd.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":14,"Channel":1,"Input":0},"ListenHEreFlicks":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\Listen Here Nwword.wav","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":39,"Channel":1,"Input":0},"FlickzOhGod":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\DangCottonPN.wav","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":49,"Channel":1,"Input":0},"minecraft sound":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\Challenge Complete (Minecraft Hero of the Village Sound) - Sound Effect for editing.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":33,"Channel":1,"Input":0},"do it":{"OBSCallBacksON":[],"OBSCallBacksOFF":[],"OBSCallBacksSlider":[],"SoundCallBack":{"File":"C:\\Users\\KlutzyBubbles\\Music\\sounds\\do-it_pnCjXeb.mp3","DeviceName":"VoiceMeeter Input (VB-Audio Voi","StopWhenReleased":true,"Loop":false,"Volume":1.0,"StopAllOtherSounds":false,"Device":4},"MediaCallBack":null,"MediaCallBackOFF":null,"TwitchCallBackON":null,"TwitchCallBackOFF":null,"MIDIControlCallBackON":null,"MIDIControlCallBackOFF":null,"GoXLRCallBackON":null,"GoXLRCallBackOFF":null,"Mididevice":"Launchpad MK2","NoteNumber":23,"Channel":1,"Input":0}}
```

**Comment by KlutzyBubbles** [comment]
I did some further troubleshooting and i think it had something to do with a program i had made a while back. I removed the program restarted the midi device and your program is back to working perfectly. Sorry to waste time on an issue i created :/

I also tried recreating the issue to actually try find the cause but no luck, if it happens again and i can provide a repeatable example i will definitely let you know but right now just seems like an odd case.
